### PR TITLE
Don't serialize args in account.ts

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

File: src/transaction.ts [hunk 1]
```diff
@@ -51,8 +51,11 @@ export function deployContract(code: Uint8Array): Action {
     return new Action({ deployContract: new DeployContract({code}) });
 }
 
-export function functionCall(methodName: string, args: Uint8Array, gas: BN, deposit: BN): Action {
-    return new Action({functionCall: new FunctionCall({methodName, args, gas, deposit }) });
+export function functionCall(methodName: string, args: Uint8Array | object, gas: BN, deposit: BN): Action {
```
Comment (janedegtiareva) [inline src/transaction.ts:54]: Do you mind adding comments to the function? Specifically a note about the possible formats for args would be very helpful. Thanks!

**Comment by janedegtiareva** [comment]
Looks like there are test failures to look at

File: src/transaction.ts (hunk 1 above)
Comment (vgrichina) [reply src/transaction.ts:63]: Done

**Comment by vgrichina** [comment]
> Looks like there are test failures to address

looks like these were just lint failures, fixed now

**Pull request merged**